packages/spec-driver-webdriverio/src/spec-driver.ts

```diff
@@ -176,7 +176,19 @@ export async function findElement(
 ): Promise<Applitools.WebdriverIO.Element> {
   selector = utils.types.has(selector, ['using', 'value']) ? `${selector.using}:${selector.value}` : selector
   const root = parent ? await browser.$(transformShadowRoot(parent) as any) : browser
-  const element = await root.$(selector)
+  let element
+  try {
+    element = await root.$(selector)
+  } catch (error) {
+    if (
+      /element could not be located/i.test(error.message) ||
+      /cannot locate an element/i.test(error.message) ||
+      /wasn\'t found/i.test(error.message)
+    ) {
+      return null
+    }
+    throw error
+  }
   return !utils.types.has(element, 'error') ? element : null
```

Comment:
I would prefer something like this, becasue it is a little easy to read, on my opinion.
```suggestion
  try {
    const element = await root.$(selector)
    return !utils.types.has(element, 'error') ? element : null
  } catch (error) {
    if (
      /element could not be located/i.test(error.message) ||
      /cannot locate an element/i.test(error.message) ||
      /wasn\'t found/i.test(error.message)
    ) {
      return null
    }
    throw error
  }
```